tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -18,4 +19,35 @@ trait Collaboration {
     Then("they should see a 'No merge requests to display' info")
     verify userCanSee projectPage.Collaboration.MergeRequests.noMergeRequests
   }
+
+  def verifyIssuesIsEmpty(implicit projectDetails: ProjectDetails): Unit = {
+    val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    And("they navigate to the Merge Requests sub tab")
+    click on projectPage.Collaboration.Issues.tab
+    Then("they should see a 'No issues to display' info")
+    verify userCanSee projectPage.Collaboration.Issues.noIssues
+  }
+
+  def createNewIssue(implicit projectDetails: ProjectDetails): Unit = {
+    val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    And("they navigate to the Merge Requests sub tab")
+    click on projectPage.Collaboration.Issues.tab
+    // Give some time for the tab change to take effect
+    sleep (1 second)
+    And("the user clicks on the 'New Issue' button")
+    click on projectPage.Collaboration.Issues.newLink
+    And("they fill out the form")
+    val issueTitle = "test issue"
+    val issueDesc = "test description"
+    projectPage.Collaboration.Issues.NewIssue.createIssueWithTitleAndDescription(issueTitle, issueDesc)
```

Comment:
Just a thought that maybe having the body of the `createIssueWithTitleAndDescription` method directly in the workflow here could be a bit better. Not so sure if with the current approach we are mixing up the responsibilities of the workflows (which I believe should be actions) with the page classes (which are page definitions). I know we took a similar approach in other places but maybe we can try to refactor these (not necessarily as part of this issue).

Reply:
I am not convinced this will improve the tests. I agree that it is debatable whether filling out a form is part of the page definition, but keeping it in the page makes the workflow easier to read, since the code that fills out the form is neither particularly interesting nor relevant to the workflow; the workflow just cares that the form gets filled out and submitted.

Reply:
We agree that we should keep workflows as simple as possible but we'd still like workflows to be actions grouping entities, not pages. So what would you think if we have simple workflows responsible for things like `createIssueWithTitleAndDescription`? And maybe we could use backticks for naming workflows `def` so they look more like sentences and help to boost readability? (I mean to have ``  `create an issue with title and description` ``  instead of `createIssueWithTitleAndDescription`)